python/doc/examples/reliability_sensitivity/reliability/plot_create_threshold_event.py

```diff
@@ -0,0 +1,88 @@
+"""
+Create a threshold event
+========================
+"""
+# %%
+# Abstract
+# --------
+#
+# We present in this example the creation and the use of a :class:`~openturns.ThresholdEvent` to estimate a simple integral.
+#
+import openturns as ot
+import openturns.viewer as otv
+from matplotlib import pylab as plt
+
+
+# %%
+# We consider a standard unit gaussian random vector :math:`X` and build a random vector from this distribution.
```

Comment:
```suggestion
# We consider a standard Gaussian random vector :math:`X` and build a random vector from this distribution.
```
"Gaussian" because "Gauss" is a proper noun. I don't think it is necessary to specify "unit": isn't it implied by "standard"?